src/constructN.js

```diff
@@ -18,18 +18,20 @@ var nAry = require('./nAry');
  * @return {Function} A wrapped, curried constructor function.
  * @example
  *
- *      // Variadic constructor function
- *      var Widget = () => {
- *        this.children = Array.prototype.slice.call(arguments);
- *        // ...
+ *      // Constructor function
+ *      function Animal(kind, sound) {
```

Comment:
if you are illustrating `constructN`, I think it makes sense to have a nullary constructor here that assigns its fields based on argument position. If There is no reason to constrain the arity of the output function, then why bother with `N`?